go/src/koding/remoteapi/models/j_group_role.go

```diff
@@ -16,7 +16,7 @@ type JGroupRole struct {
 	// id
 	ID string `json:"_id,omitempty"`
 
-	// is configurable
+	// is configureable
```

Comment:
@rjeczalik whe have three options:

1) Fix it in jRole (which is most complicated)
2) Alter generated file by hand (which is dummy)
3) Remove spelling check from gometallinter (which is sad)

Personally, I don't think it's important, just pointing that tests failed because of this. If `1)` is too complicated we may end up with `2)`